common/persistence/nosql/nosqlplugin/cassandra/tasks.go

```diff
@@ -110,7 +110,7 @@ func fromTaskListPartitionConfig(config *persistence.TaskListPartitionConfig) ma
 // InsertTaskList insert a single tasklist row
 // Return TaskOperationConditionFailure if the condition doesn't meet
 func (db *cdb) InsertTaskList(ctx context.Context, row *nosqlplugin.TaskListRow) error {
-	timeStamp := db.timeSrc.Now()
+	timeStamp := row.LastUpdatedTime
```

Comment:
insert is a create operation. It would be confused to have lastUpdatedTime. I understand that you might want to reuse the LastUpdateTime in TaskListRow struct, but maybe rename it to like "timestamp" would be a good idea.

Reply:
I suggest using a single expression, like `currentTimestamp`, for all functions (insert, update). What do you think?